### PR TITLE
chore: update changelog

### PR DESCRIPTION
**Pull request by csm-thu**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,63 @@
...
+
+### Bug Fixes
+
+- force date values received from \(and sent to\) back\-end to be set to midnight UTC ([987f1cf](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/987f1cfb))
+- rename \.xlsx files imported in Table to \.csv before upload ([5e75dfc](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/5e75dfc0))
+- \[PROD\-9531\] prevent aggrid from automatically renaming table columns ([9fcb439](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/9fcb4390))
+- \[PROD\-11247\] make email & roles non\-case\-sensitive ([25ececb](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/25ececbd))
+- \[PROD\-12144\] make running state more visible ([d07c6bc](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/d07c6bcd))
+- sligthly increase size of run template name in Scenario view ([1c18af0](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/1c18af02))
+- \[PROD\-7940\] sort scenarios in dropdown list, in scenario creation pop-up ([9d23168](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/9d231682))
```
Comment (esasova) [inline CHANGELOG.md:38]: I'd propose to rephrase this line to be more clear about fix:
_add parent-child indentation in scenario dropdown list, in scenario creation pop-up_

The verb _sort_ can be confusing as we are not actually sorting the list but adding visual indentation :slightly_smiling_face:

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,63 @@
...
+- disable folding of organization Accordion in workspaces selector ([834f557](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/834f5576))
+- \[PROD\-12104\] running scenarios are now stopped before being deleted ([2bae6fc](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/2bae6fc0))
+- show error in browser console when conversions of parameter values have failed ([08ff30f](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/08ff30f0))
+- fix possible errors in Powerbi reducer & saga ([fcd1929](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/fcd1929d))
+- fix broken behavior of REACT_APP_NO_POWERBI_POLLING option ([be87a09](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/be87a093))
+- and more minor bug fixes...
 
 ### Known issues
 
 - clearing the errors panel of a Table parameter does not reset the "dirty" state of the scenario parameters form
```
Comment (nborde-CSM) [inline CHANGELOG.md:49]: Maybe we should reformulate the behavior issue to be more factual : "Clearing the errors panel of a table parameter causes systematically dirty state as true".

File: CHANGELOG.md (hunk 1 above)
Comment (csm-thu) [reply CHANGELOG.md:38]: I already rephrased the commit message but your version is actually way better, thanks ! :smile:

**Review by nborde-CSM — APPROVED** [review]
Two remarks but LGTM :+1:

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,8 +1,63 @@
...
+- improvements for **table** scenario parameters
+  - add **fullscreen** support for an improved user experience when working with tables
+  - when option `canChangeRowsNumber` is set, users will be able to manually **create and delete rows** in the table
+  - add support for **groups of columns**, with optional columns "collapse & expand"
+- improve checks & error messages to help integrators:
+  - show warnings in console when solution contains unrecognized keys
+  - show warnings in console when workspace contains unrecognized keys
+  - improved error messages context & format for PowerBI service account errors
+- more context shown to end-users:
+  - workspace information is now visible from scenario view
```
[the remark this answers is not in the document]
Comment (csm-thu) [reply CHANGELOG.md:21]: "refacto" is a bit too technical here. but I'll change this line :+1:

File: CHANGELOG.md (hunk 2 above)
Comment (csm-thu) [reply CHANGELOG.md:49]: As discussed, keeping this version for now :slightly_smiling_face:

**Pull request merged**